AssettoServer/Network/Http/ContentManagerController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.IO;
+using AssettoServer.Server.CMContentProviders;
+using AssettoServer.Server.Configuration;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AssettoServer.Network.Http;
+
+[ApiController]
+public class ContentManagerController : ControllerBase
+{
+    private readonly ACServerConfiguration _configuration;
+    private readonly ICMContentProvider _contentProvider;
+
+    public ContentManagerController(ACServerConfiguration configuration, ICMContentProvider contentProvider)
+    {
+        _configuration = configuration;
+        _contentProvider = contentProvider;
+    }
+
+    [EnableCors("ServerQueryPolicy")]
```

Comment:
you can remove the CORS policy from all methods in this class, it's only used for querying server status